python/setup_project/default_config_page.py

```diff
@@ -47,7 +47,8 @@ def validatePage(self):
         wait.show()
         QtGui.QApplication.instance().processEvents()
         try:
-            wiz.set_config_uri(uri)
+            # Download/validate the config. prep storage mapping display
```

Comment:
The old logic was was a bit tricky. It called `set_config_uri` on the wizard before the storage page(s) were shown. There was logic in the wizard to defer actually setting the uri until the storages were valid, offloading that responsibility on the last storage page. Now that we're always showing the storage mapping page, we can simplify the logic. Now the config pages simply "validate" the uri (which sets up the storage mapping page) and assume the storage mapping page will set the uri when everything is validated.